theano/gpuarray/opt.py

```diff
@@ -384,6 +386,8 @@ def apply(self, fgraph):
 
             if new_ops:
                 node_created[lopt] += len(theano.gof.graph.ops([mapping[i] for i in node.inputs], outputs))
+                self.new_opts.append(lopt)
+                time_opts[lopt] = t_opt2 - t_opt
```

Comment:
@nouiz is doing this correct ? I have commented this before, but wouldn't it update the time_opts dict with consecutive node's optimizer's value if they seem to use the same local optimizer ?